threedi_modelchecker/simulation_templates/models.py

```diff
@@ -183,15 +181,98 @@ async def save_to_api(self, client: V3BetaApi, simulation: Simulation):
                     simulation_pk=simulation.id, data=self.constant_gw
                 )
             )
-        if self.predefined_1d is not None:
+        initial_waterlevels: List[InitialWaterlevel] = []
+        rasters_lookup: Dict = {}
+
+        # TODO: support user uploaded 1d/2d/gw initial waterlevel file/rasters to threedimodels.
+
+        if (
+            self.raster_2d is not None
+            or self.raster_gw is not None
+            or self.waterlevel_1d_file is not None
+        ):
+            # Fetch initial waterlevels
+            initial_waterlevels = (
+                await client.threedimodels_initial_waterlevels_list(
+                    simulation.threedimodel_id
```

Comment:
Do we need to order by something here? Or is that the default of the API?

Reply:
For the amount of possible initial waterlevels, this is not needed right now.